src/app/components/fee-row/components/transaction-fee.tsx

```diff
@@ -8,7 +8,7 @@ export function TransactionFee(props: TransactionFeeProps): JSX.Element | null {
   const { fee, isSponsored } = props;
   return (
     <span data-testid={TransactionSigningSelectors.FeeToBePaidLabel}>
-      {isSponsored ? '🎉 sponsored' : fee} STX
+      {isSponsored ? '(sponsored) 0' : fee} STX
```

Comment:
Not a fan of brackets. Could we change the whole row to something else? 

Not sure what, exactly, but something that clearly indicates its free.

Reply:
Yep, wasn't too sure so threw this out there and was hoping for feedback. I know people didn't love the emoji so was attempting to remove it. Open to suggestions from @eugeniadigon or anyone with an better idea.

Reply:
My suggestion would be to remove the fee row altogether, and in its place adds a message that clearly explains "This transaction is sponsored, so no fee is charged", in a box like this, but with the blue as it isn't a warning. 

![image](https://user-images.githubusercontent.com/1618764/150806945-2fd223ae-b70a-4fcf-9f2a-4893bb53dd93.png)